src/mam4xx/aero_model.hpp

```diff
@@ -174,8 +183,9 @@ void calc_rain_drop_conc(const int nr, const Real rlo, const Real dr,
     precipsum += vfallrainsv[ii] * rr * rr * rr * xnumrainsv[ii];
 
   } // ii
-    // 1.333333 is simplified 4/3 for sphere volume calculation
-  precipsum *= haero::Constants::pi * Real(4) / Real(3);
+
+  // 1.333333 is simplified 4/3 for sphere volume calculation
```

Comment:
This comment no longer applies now that we have `four_thirds`, right?

Reply:
Yes.